python/rpdk/python/codegen.py

```diff
@@ -157,6 +158,10 @@ def generate(self, project):
 
         LOG.debug("Generate complete")
 
+    # pylint: disable=unused-argument
```

Comment:
suppressing this warning here since in Java plugin we use the project reference to fetch plugin runtime version from pom.xml.

Reply:
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/349/files#diff-16dc52f25a203c93708a8ce01a72cefce64b026df8f5a8ec8a16d737d10d3179R587